### PR TITLE
Update java sdk version

### PR DESCRIPTION
**Pull request by AsafMah**
Update the java version to a stable one, which will be fetched using bundle.

### CODE REVIEW

**Review by tamirkamara — CHANGES_REQUESTED** [review]
Nice to see the progress on handling jars directly.
I might be missing something though - why do you need to add all the jars themselves to the project? can't those be downloaded in the build process?

File: lib/logstash/outputs/kusto/ingestor.rb [hunk 1]
```diff
@@ -9,8 +9,7 @@ class LogStash::Outputs::Kusto < LogStash::Outputs::Base
   # This handles the overall logic and communication with Kusto
   #
   class Ingestor
-    require 'kusto/kusto-ingest-1.0.0-BETA-04-jar-with-dependencies.jar'
-
+    require 'logstash-output-kusto_jars'
```
[the remark this answers is not in the document]
Comment (AsafMah) [reply lib/logstash/outputs/kusto/ingestor.rb:12]: This file is autogenerated by jar-dependencies, I don't really have control over the name sadly

File: .travis.yml [hunk 2]
```diff
@@ -3,13 +3,13 @@ language: ruby
 cache: bundler
 matrix:
   include:
-  - rvm: jruby-9.1.13.0
+  - rvm: jruby-9.2.13.0
     env: LOGSTASH_BRANCH=7.0
```
[the remark this answers is not in the document]
Comment (AsafMah) [reply .travis.yml:9]: I added one for 7.x, so we'll always have the latest version

**Comment by AsafMah** [comment]
> Nice to see the progress on handling jars directly.
> I might be missing something though - why do you need to add all the jars themselves to the project? can't those be downloaded in the build process?

They are downloaded automatically when doing bundle install,
I commited them because the jar was commited before.
You don't think it's right?

**Comment by tamirkamara** [comment]
@AsafMah In the past, Jars needed to be here since they weren't available in maven at the time. This is no longer the case, and if the build process/script is able to download everything required - I think those shouldn't be in here...

**Comment by AsafMah** [comment]
Travis passed - https://travis-ci.org/github/Azure/logstash-output-kusto/builds/749998370

I'm not sure why it doesn't display properly

**Review by tamirkamara — APPROVED** [review]

**Pull request merged**